src/Containers/packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -325,17 +330,9 @@
           ContainerRegistry=$(ContainerRegistry);
           _ContainerImageTags=@(ContainerImageTags, ';');
           ContainerRepository=$(ContainerRepository);
-          ContainerWorkingDirectory=$(ContainerWorkingDirectory);
-          _ContainerEntrypoint=@(ContainerEntrypoint, ';');
-          _ContainerEntrypointArgs=@(ContainerEntrypointArgs, ';');
-          _ContainerAppCommand=@(ContainerAppCommand, ';');
-          _ContainerAppCommandArgs=@(ContainerAppCommandArgs, ';');
-          ContainerAppCommandInstruction=$(ContainerAppCommandInstruction);
-          _ContainerDefaultArgs=@(ContainerDefaultArgs, ';');
           _ContainerLabel=@(ContainerLabel->'%(Identity):%(Value)');
           _ContainerPort=@(ContainerPort->'%(Identity):%(Type)');
           _ContainerEnvironmentVariables=@(ContainerEnvironmentVariable->'%(Identity):%(Value)');
-          ContainerUser=$(ContainerUser);
```

Comment:
Could you please double check that the items and properties set globally are still available inside `_PublishSingleContainer` in multi-arch scenario after deleting these lines?

Reply:
Yep! Here's a screenshot from a multi-arch build I did locally:

![image](https://github.com/user-attachments/assets/8c75aa03-b0b9-4dec-a4df-88133f004a16)

In this case, `ContainerUser` was set in the project file directly, as was `ContainerAppCommandArgs`, and both appear in the `CreateNewImage` parameters as expected.